README.md

```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
+   section lists the files that have been modified by the generator. When
+   clicking on a file, VS Code shows side-by-side the current and updated
+   version of the file. Changes can be accepted or rejected at the level of an
+   entire file or selected lines.
 
-    npx @openapitools/openapi-generator-cli \
-        generate -i dist.yaml -g python-flask -o server
+### Testing the NLP tool
 
-Then compare the ~/nlp-sandbox-data-annotated-example-updated/server to your existing ~/nlp-sandbox-data-annotated-example/server directory to see
-what was updated.
+This command will check that your Python code adheres to the style guide defined
+for this project (see [server/setup.cfg](server/setup.cfg)).
 
-The other method, once you are more confident, is to lay the files on top of the existing repository you've already checked with the command:
+    npm run lint
 
-    openapi-generator generate -i dist.yaml -g python-flask -o ~/nlp-sandbox-data-annotated-example/server
+The command below will run the unit and integration tests of this project.
 
-If one wants to prevent certain files you know have already been customized then add those file names
-cto the ~/nlp-sandbox-data-annotated-example/server/.openapi-generator-ignore file before running the preceding command.
+    npm run test
 
-Then use git to see what is updated and if you overwrote any files you wanted
-to preserve. One can revert those changes and add those files to the .openapi-generator-ignore file for next time there is an update. -->
+### Versioning
 
+This package uses [semantic versioning] for releasing new versions. Creating a
+GitHub release of this repository will trigger the CI/CD workflow which will in
+turn build the new Docker image for this tool before publishing it to DockerHub.
 
-### Generate a Spring Boost server stub
+We recommend to include the following information at the top of your README:
 
-Generate the initial server stub from the OpenAPI specification
+- `Date Annotator API version`: The version of the Date Annotator specification
+  implemented by the tool.
+- `Tool version`: The version of the tool.
 
-    mkdir server
-    ./node_modules/.bin/openapi-generator-cli \
-        generate -i openapi.yaml -g spring -o server
+Initially, it may be tempating to align the tool version to the API version. As
+you improve your tool and fixes bugs, you will likely release more than one
+version of your tool for the same API version. Note that for the sake of
+reproducibility, it is encouraged to not reuse version tags.
 
-Build and start the server with Maven
+## Benchmarking
 
-    cd server
-    mvn package
-    java -jar target/openapi-spring-0.1.6.jar
+Visit nlpsandbox.io for instructions on how to submit your tool and evaluate its
+performance on public and private datasets.
 
-The API documentation UI is now available at http://localhost:8080.
+## License
 
+[Apache License 2.0]
 
-<!-- Definitions -->
+<!-- Links -->
```

Comment:
Not sure if this is on purpose but this section is still hidden from view.

Reply:
Yes. Everything below this section is not visible.